functions/AFQ_SegmentFiberGroups.m

```diff
@@ -156,8 +156,17 @@
 % Set the directory where templates can be found
 tdir = fullfile(fileparts(which('mrDiffusion.m')), 'templates');
 % Initialize spm defualts for normalization
-spm_defaults; global defaults; params = defaults.normalise.estimate;

```

Comment:
If it is working for you just ignore the change please. First of all I wanted to make sure that it was working in my system and I had to do some changes. This was tested on OSX and now I am moving everything to the cluster with Red Hat 6.